stack-8.4.4.yaml

```diff
@@ -12,7 +12,7 @@ extra-deps:
 - bifunctors-5.5.6
 - brittany-0.12.1.1
 - bytestring-trie-0.2.5.0
-- Cabal-3.0.0.0@sha256:1ba37b8d80e89213b17db7b8b9ea0108da55ca65f8c0cbb7433881a284c5cf67,26027
+- Cabal-2.4.1.0
```

Comment:
This does not seem correct?

Reply:
Stack recommended to set cabal version explicitly in https://circleci.com/gh/haskell/haskell-ide-engine/13311
If this is incorrect then I don't think I know how to proceed with this

Reply:
Does this compile? Because, iirc, stack snapshots are bundled with `cabal-install` and `Cabal` versions, iirc, setting a custom Cabal version for a snapshot may not succeed. If it compiles, everything is fine.

Reply:
Not yet, I'm slowly bisecting towards the cabal version satisfying every dependency

Reply:
And back to stack suggesting `Cabal-3.0.0.0`. Looks like stylish `0.10.0.0` can't be used here and below because of conflicting dependencies :-(